packages/data-context/test/unit/actions/project/LoginForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
```

Comment:
Could trim this component down to just what was causing the parsing error, if we add more of these edge-case tests it would be nice to not bloat the files.

Reply:
Yeah good point. I trimmed basically everything except the interface definition since that's what was causing the syntax error originally